compiler/fir/checkers/gen/org/jetbrains/kotlin/fir/analysis/checkers/declaration/ComposedDeclarationCheckers.kt

```diff
@@ -49,14 +49,14 @@ internal class ComposedDeclarationCheckers : DeclarationCheckers() {
 
     @CheckersComponentInternal
     internal fun register(checkers: DeclarationCheckers) {
-        _basicDeclarationCheckers += checkers.allBasicDeclarationCheckers
-        _memberDeclarationCheckers += checkers.allMemberDeclarationCheckers
-        _functionCheckers += checkers.allFunctionCheckers
-        _propertyCheckers += checkers.allPropertyCheckers
-        _classCheckers += checkers.allClassCheckers
-        _regularClassCheckers += checkers.allRegularClassCheckers
-        _constructorCheckers += checkers.allConstructorCheckers
-        _fileCheckers += checkers.allFileCheckers
+        _basicDeclarationCheckers += checkers.basicDeclarationCheckers
+        _memberDeclarationCheckers += checkers.memberDeclarationCheckers
+        _functionCheckers += checkers.functionCheckers
+        _propertyCheckers += checkers.propertyCheckers
+        _classCheckers += checkers.classCheckers
+        _regularClassCheckers += checkers.regularClassCheckers
+        _constructorCheckers += checkers.constructorCheckers
+        _fileCheckers += checkers.fileCheckers
```

Comment:
This file and below are automatically generated, but I don't see any counterpart change in the generator?

Reply:
Oops! I missed that it's auto-generated 😛